### ISSUE TITLE
Crossrunner seems to be destroying 5 1/4" floppy images

### ISSUE THREAD

**Issue by g051051** [ibrumby/Crossrunner-Issues#28]
Every time I try to work with 5 1/4" disk images, they seem to get corrupted.  For instance, running GS/OS 6.0.4 with the DOS 3.3 FST, disks will mount and be readable, but will get corrupted when ejecting or shutting down the emulated system.  Booting games the write back to the disk (or any disk) will corrupt the disk as well.  RobotWar won't create a robot storage disk no matter what I do.  Running a battle will also eventually corrupt the disk when it writes the battle info or tries to update the scores, so that the image won't run after a reboot.

**Comment by GBirkel** [comment]
I had a similar issue in OpenEmulator with .woz format images.  Any time I needed to swap between disks (eject one, insert another) after writing data there was a more than 50% chance the first disk would be completely unreadable the next time I put it in.  Have you seen any pattern with image formats?

**Comment by g051051** [comment]
> I had a similar issue in OpenEmulator with .woz format images. Any time I needed to swap between disks (eject one, insert another) after writing data there was a more than 50% chance the first disk would be completely unreadable the next time I put it in. Have you seen any pattern with image formats?

No, I've been using .dsk files from Internet Archive (like RobotWar).  I also created some .do, and .d13 images with CiderPress2 (when I was trying to create Robot storage disks for RobotWar).

**Comment by ibrumby** [comment]
This should be fixed in the next version.
